src/handlers/records-query.ts

```diff
@@ -75,19 +75,43 @@ export class RecordsQueryHandler implements MethodHandler {
     return recordsQueryReplyEntries;
   }
 
+  /**
+   * Convert an incoming DateSort to a sort type accepted by MessageStore
+   * Defaults to 'dateCreated' in Descending order if no sort is supplied.
+   *
+   * @param dateSort the optional DateSort from the RecordsQuery message descriptor.
+   * @returns {MessageSort} for MessageStore sorting.
+   */
+  private convertDateSort(dateSort?: DateSort): MessageSort {
+    switch (dateSort) {
+    case DateSort.CreatedAscending:
```

Comment:
No action item to this PR, but seems like the DWN specification is missing `dateModified`, which would be sorted against `messageTimestamp`

Reply:
I updated the code according to my comment, see if you like/hate it.

Reply:
💯 That is definitely a useful sort to expose.